src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy

```diff
@@ -16,5 +16,6 @@ class InspectContainerResponseHandler implements ResponseHandler<String, Object>
         logger.quiet "VolumesFrom : $container.hostConfig.volumesFrom"
         String exposedPorts = container.config?.@exposedPorts ? container.config.exposedPorts : '[]'
         logger.quiet "ExposedPorts : $exposedPorts"
+        logger.quiet "LogConfig : $container.hostConfig.logConfig.type.type"
```

Comment:
Is `type.type` correct? Shouldn't it just be `$container.hostConfig.logConfig.type`?